frontend/src/scenes/pipeline/Pipeline.tsx

```diff
@@ -0,0 +1,17 @@
+import { SceneExport } from 'scenes/sceneTypes'
+import { pipelineLogic } from './pipelineLogic'
+
+export function Pipeline(): JSX.Element {
+    return (
+        // TODO: consolidate on a recommended naming convention
+        <div className="pipeline-scene">WIP: Pipeline Scene!</div>
```

Comment:
Suggestion: you can start with a `PageHeader`, since that's the bare minimum of a scene